src/doc/DocDrawingThread.js

```diff
@@ -147,6 +147,15 @@ class DocDrawingThread extends DrawingThread {
         return !!(location && !!this.location && !!this.location.page && this.location.page !== location.page);
     }
 
+    /** @inheritdoc */
+    destroy() {
+        if (this.drawingContext) {
+            this.drawingContext.canvas.remove();
```

Comment:
Oh, can you double check that there's a polyfill in place, where this is used? `remove()` is unsupported in IE